src/main/java/com/exactpro/th2/codec/csv/CsvCodec.java

```diff
@@ -48,18 +49,18 @@
 import com.exactpro.th2.common.value.ValueUtils;
 import com.google.protobuf.ByteString;
 
-public class CsvCodec implements IPipelineCodec {
+public abstract class CsvCodec implements IPipelineCodec {
```

Comment:
Are we going to create more then one implementation? If protobuf and transport implementation haven't got common logic we can create th2 separate classes without inheritance to split code otherwise please put logic into one CsvCodec class